README.md

```diff
@@ -1 +1,27 @@
-# helm-charts
\ No newline at end of file
+# Volcano Community Kubernetes Helm Charts
+
+The code is provided as-is with no warranties.
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```console
+helm repo add volcano-sh https://volcano-sh.github.io/helm-charts
+```
+
+You can then run `helm search repo volcano-sh` to see the charts.
+
+
+**Install Volcano:**
```

Comment:
Is there any possible to specify the version?

Reply:
When `helm install`, you can use `--version` to specify the version name, for example: "--version 1.6.0", if not specified, the latest version will be installed by default.

Reply:
As far as I understand, there is no need to specify the version number here, just use the latest version to install it.